code-challenges.test.js

```diff
@@ -47,21 +107,110 @@ const studyMinutesWeek2 = {
+  tuesday: 20,
+  wednesday: 15,
+  thursday: 30,
+  friday: 15,
+  saturday: 60
+}
+const studyMinutesWeek2 = {
+  sunday: 100,
+  monday: 10,
+  tuesday: 45,
+  wednesday: 60,
+  thursday: 20,
+  friday: 15,
+  saturday: 65
+}
+expect(sortedObjs(studyMinutesWeek1)).toEqual( [15, 15, 20, 30, 30, 60, 90] );
+expect(sortedObjs(studyMinutesWeek2)).toEqual( [10, 15, 20, 45, 60, 65, 100] );
+})})
+
+const sortedObjs = (obj) => {
+  var numVal = Object.values(obj);
+  return numVal.sort((a , b) => a - b)
+}
+
 
-const accountTransactions1 = [100, -17, -23, -9]
-// Expected output: [100, 83, 60, 51]
+console.log(sortedObjs(studyMinutesWeek1)) OUTPUT: [ 15 , 15 , 20 , 30 , 30 , 60 , 90 ]
+console.log(sortedObjs(studyMinutesWeek2)) OUTPUT:[ 10 , 15 ,  20 , 45 , 60 , 65 , 100]
 
-const accountTransactions2 = [250, -89, 100, -96]
-// Expected output: [250, 161, 261, 165]
+PASS  ./code-challenges.test.js
+sortedObjs
+*/
+// ********************************************************************************************************************************
+
+/*
+
+// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+*/
+// ********************************************************************************************************************************
+/*
+INPUT: TAKES IN AN ARRAY 
+OUTPUT: RETURNS AN ARRAY OF THE ACCUMULATING SUM, AN EMPTY ARRAY SHOULD RETURN AN EMPTY ARRAY
+METHODS: ACCUMULATING SUM REFERENCES TO SOME SORT OF MATH ((A,B) => A + B). ITERATION THROUGH AN ARRAY OF VALUES NEEDS TO OCCUR. 
+*/
+// ********************************************************************************************************************************
+/*
+ a) Create a test with expect statements for each of the variables provided.
+
+describe(`acSum`,  () => {
+  it(`RETURNS AN ARRAY OF THE ACCUMULATING SUM, AN EMPTY ARRAY SHOULD RETURN AN EMPTY ARRAY`, () => {
+    const accountTransactions1 = [ 100 , -17 , -23 , -9 ]
+    const accountTransactions2 = [ 250 , -89 , 100 , -96 ]
+    const accountTransactions3 = [ ]
+    expect(acSum(accountTransactions1)).toEqual([ 100 , -17 , -23 , -9 ]);
+    expect(acSum(accountTransactions2)).toEqual([ 250 , -89 , 100 , -96 ]);
+    expect(acSum(accountTransactions3)).toEqual([  ]);
+
+  })
+})
+
+ReferenceError: acSum is not defined ---->  FAIL  ./code-challenges.test.js
+acSum
 
-const accountTransactions3 = []
-// Expected output: []
 
 // b) Create the function that makes the test pass.
+
+
+describe(`acSum`,  (arr) => {
+  it(`RETURNS AN ARRAY OF THE ACCUMULATING SUM, AN EMPTY ARRAY SHOULD RETURN AN EMPTY ARRAY`, () => {
+    const accountTransactions1 = [ 100 , -17 , -23 , -9 ]
+    const accountTransactions2 = [ 250 , -89 , 100 , -96 ]
+    const accountTransactions3 = [ ]
+    expect(acSum(accountTransactions1)).toEqual([ 100 , -17 , -23 , -9 ]);
+    expect(acSum(accountTransactions2)).toEqual([ 250 , -89 , 100 , -96 ]);
```

Comment:
Your expected output is not correct, make sure you pay attention to what the prompt is telling you the function should output.